Miscellaneous Coalition Missions (#4934)
endless-sky/endless-sky: data/coalition/coalition missions.txt

	name "Transport Chikee"
	description "Chikee, a young Kimek, is studying to become a Heliarch. Transport him to <destination>, where he was offered a job, so that he can make a living while studying on his own."
	passengers 1
	to offer
		has "Coalition: First Contact: done"
	source "Fourth Shadow"
	destination "Second Rose"
	on offer
		conversation
			`At any given time there are thousands of Kimek here in the spaceport, either mingling among themselves during a break from work, or negotiating travel prices with starship captains. You imagine that if you weren't a human and clearly stood out, you too would be constantly getting offers to travel to another world, as many civilian captains here wish to fill a last bunk or two they have vacant on their ship before departing.`
			`	You manage to find a restaurant here where you spot some food you've already learned you can eat, so you take a seat and order something.`
			`	You're halfway through your meal when you notice the waiter who brought it to you still hasn't moved away from the table.`
			choice
				`	"Can I help you?"`
				`	"Sorry, should I have paid first before eating?"`
			branch translator
				has "language: Coalition"

> 			`	The Kimek doesn't have a translation box, so your words don't reach him, and while he insists in remaining by the table, you also notice he has been constantly glancing at the entrance, as if waiting for someone to come in.`

Reviewer: Fairly sure it should be insists on not insists in

			`	When a Heliarch agent passes by the entrance, the Kimek says something in his language, heading to meet the Heliarch, and speaking with them briefly. The Heliarch agent then accompanies the Kimek waiter to your table.`
			choice

> 				`	"Is there an officer problem?"`

Reviewer: Is there a problem officer (Or is it intentional?)

Reviewer 2: you could suggest a commit, you know. makes it easier on him.

Author: was intentional as a joke

	name "Transport Chikee"
	description "Chikee, the young Kimek who you've transported many times, has finally become a candidate for elevation to the rank of Heliarch. Bring him to <destination> by <date>."
	deadline
	landing
	passengers 1
	to offer
		has "event: chikee break 3"
	source "Ablub's Invention"
	destination "Ring of Wisdom"
	on offer
		conversation
			`You're contacted by Chikee when you land.`
			`	"Captain <last>! Good news, and another job for you, I have," he says, asking that you meet him in the spaceport.`
			`	You head there, and he's already with his bags, prepared to board your ship. "Helped me a lot, House Idriss has. An audience with the Heliarch, on <destination> I have!" he says.`
			`	He says that he needs to be there by <date>, and wastes no time as he immediately heads into your ship.`
				accept
	on complete
		"coalition jobs" ++
		payment 300000
		conversation
			`Chikee hands you <payment> before leaving your ship. Heliarch agents are waiting for him, and receive him in a bit of a ceremony.`
			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`

> 			`	Rinworlds have no day or night, but looking at the conventioned time used here, it is almost night time when Chikee returns to your ship, in a hurry.`

Reviewer: Ringworlds not Rinworlds, Not sure if conventioned is right.

Author: suggestion for a word that would be better than conventioned, given the context?

Reviewer: Maybe change it to be based off your ship's internal clock as your reference?

> 			`	"Accepted I was! Grant me the rannk, they did!" he says, showing you a circlet, a Heliarch circlet, as he puts it on his head. "Infinitely grateful for your help in all of this I am, Captain."`

Reviewer: Rank not rannk

> 			`	You congratulate him on the achievement, and wish him best of luck in his new line of work, as he leaves your ship one last time.`

Reviewer: wish him *the* best of luck

			`	"Fairly new to us, this virus is," the Kimek manning the stand says. "But finally developed an initial batch of vaccines for it, the researchers on Secret Sky thankfully have."`
			choice
				`	"What do you mean "finally"? Didn't this virus just break out recently?"`
				`	"You want me to go there retrieve the vaccines?"`
					goto vaccines
			`	He looks puzzles by your question.`
			`"That some weeks have already passed, true it is, but hard at work, the researchers have been," he says. "Blame them for taking a few days more than usual, we cannot. Intricate, this virus is."`
			`	However unaware they may be of it, clearly the Coalition's standards for public health are much more advanced than in other parts of the galaxy you've visited.`
			label vaccines
			`	He shows you to a detailed map of Kimek space, explaining that most of the confirmed and suspected cases have been there, as one would expect given that the Kimek population is more than double that of the Arachi and the Saryds combined.`
			`	"To <destination>, this initial batch must be brought. From there, distributing more batches of the vaccines to other worlds, you will be tasked with."`

> 			`	As the line of volunteering merchant captains continues to grow behind you, he is switft to inform you that payment will be <payment>, and asks that the next person come forward.`

Reviewer: swift not switft

				accept
	on stopover "Secret Sky"
		dialog `A group of Saryd workers is waiting for you and they load the <cargo> onto your ship. A laboratory worker babbles about how you must care for them.`
	on visit
		dialog `You've reached <planet>, but the vaccines aren't here yet. Depart and wait for whatever ships are carrying them to arrive.`
	on complete
		payment 943720
		"coalition jobs" ++
		conversation
			`An incessant stream of freighters has the landing pads of the spaceport here overcrowded, and from the looks of it you only got a quick clearance due to carrying the vaccines.`

> 			`	The people here who are surprisingly managing to keep the place organized despite the overwhelming traffic pay you <payment>, as they unload the vaccines off your cargo hold.`

Reviewer: Sentence flow here feels a bit off, feels like some commas might be needed.

Author: is it better now?

Reviewer: I was thinking more along the lines of "The people here, who are surprisingly managing to keep the place organized despite the overwhelming traffic, pay you <payment> as they unload the vaccines off your cargo hold."

Author: that's... what I did yes. think I can take that as a yes then? :v

Reviewer: Sorry, appears that the version I read wasn't the updated one. Ye, it's good now.

			`	His colleagues shake their heads, "Shipping here, the Arachi no longer are."`
			`	Noticing you're still there, the Saryd decides to ask, "Human, a merchant Captain you are, correct? Looking for work, you are?"`
			choice
				`	"What kind of work?"`
					goto work
				`	"Oh no, I live here actually. In the humantown district, right around the corner."`
				`	"Actually I'm just here for the view, sorry."`
					decline
			`	They all start laughing, some clearly finding it much funnier than you anticipated. They happen to be the same ones from whom you notice the stronger scent of alcohol.`
			`	"Fair enough. Not a common sight here, humans are, Captain," the one with a translation box says. "Asking for such a response, I was."`
			label work
			`	They tell you that they are students at Starlit University, and will be graduating soon. They intend on throwing a party to celebrate, and had the idea of buying some traditional Arachi Rum for that.`
			`	"Stopped shipping it here, the Arachi have," another Saryd says, pulling on the translation box on his friend's neck, nearly yanking it off. "Happened with many other drinks, this also has."`

> 			`	They explain that the rum is produced on <destination>, and that you'll probably have an easier time finding it there. Though you can't know for sure how many people they intend on inviting to this party, and although you don't know how much of a drink a Saryd can have, you still worry a bit when they say that "two tons should do."`

Reviewer: "How much a saryd can drink" might be the better way of phrasing that.

			`	"Remained stable, the Saryd population has," one of them comments. "At about 30 billion individuals, our numbers still are, though found a very slight increase in fertility rate has been."`
			`	As you continue to chat, one of them gets curious and asks you about humanity's population, to which you respond that your species numbers in about 100 billion people.`
			`	"Double that of the Arachi," one of the workers says.`
			`	"And half that of the Kimek," another one completes.`
			`	They ask you some more questions about human territory, but avoid getting too technical to the point where it's difficult for you to respond.`
				goto end
			label shops
			`	You try and entertain yourself by looking at the shops here, but pretty much all of them are just filled with the produce of the farms here.`
			`	One of them has a few toys made out of straw, you take a look at the Saryd designs for them, though they're nothing particularly special, nor as stereotypical as one might have expected from a species that resembles centaurs. Most of them appear to be replicating creatures of the local fauna, or buildings like small houses.`

> 			`	You notice two other census workers have come back, so you decide to return to the group and wait for the final member. Some Coalition captains from early have gathered together in a bar to lament the long wait.`

Reviewer: From earlier

	on offer
		conversation
			`You're contacted by Chikee when you land.`
			`	"Captain <last>! Good news, and another job for you, I have," he says, asking that you meet him in the spaceport.`
			`	You head there, and he's already with his bags, prepared to board your ship. "Helped me a lot, House Idriss has. An audience with the Heliarch, on <destination> I have!" he says.`
			`	He says that he needs to be there by <date>, and wastes no time as he immediately heads into your ship.`
				accept
	on complete
		"coalition jobs" ++
		payment 300000
		conversation
			`Chikee hands you <payment> before leaving your ship. Heliarch agents are waiting for him, and receive him in a bit of a ceremony.`
			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`
			`	Ringworlds have no day or night, but looking at your ship's internal clock, it is almost night time when Chikee returns to your ship, in a hurry.`
			`	"Accepted I was! Grant me the rank, they did!" he says, showing you a circlet, a Heliarch circlet, as he puts it on his head. "Infinitely grateful for your help in all of this I am, Captain."`

> 			`	You congratulate him on the achievement, and wish him the best of luck in his new line of work, as he leaves your ship one last time.`

Author: wdym? the last paragraph?

> 			`	You congratulate him on the achievement, and wish him the best of luck in his new line of work, as he leaves your ship one last time.`

Reviewer 3: To me, it feels like, for a mission chain a few missions long, it should have a little bit more of a resolution. As of now, you congratulate him and say bye for the last time all in one sentence. For the ending of a mission chain, this seems too short to me.

> 			`	You congratulate him on the achievement, and wish him the best of luck in his new line of work, as he leaves your ship one last time.`

Author: I've added some stuffs to it, would you say it is better now?

> 			`	You congratulate him on the achievement, and wish him the best of luck in his new line of work, as he leaves your ship one last time.`

Reviewer 3: Yep, that’s definitely better.

			`	The line moves quickly, and once it's your turn, you're paid <payment>.`
			`	Looking back at the line, and seeing the sheer number of captains waiting for their share, it's no wonder the Kimek that paid you were extremely brief, but as you come back to your ship, you see that a message was sent to you by the Coalition government. More specifically, it appears to have been sent by their Department of Health.`
			`	"To all captains who helped in dealing with the recent viral outbreak, you have our gratitude, and may rest tonight knowing you've contributed to the continued prosperity of our Coalition."`

mission "Saryd Students 1"
	minor
	name "Acquire Arachi Rum"
	description "Some Saryd students on <origin> have offered to pay you for bringing them a shipment of traditional Arachi Rum, currently only sold on <destination>."
	deadline
	to offer
		has "Coalition: First Contact: done"
	source "Shadow of Leaves"
	destination "Weir of Glubatub"
	on offer
		conversation
			`A young Saryd bumps into you as you're passing by a nearby shop, knocking you to the ground.`

> 			`	They help you up, apologizing. A couple that were with him entered the shop without even noticing you, but almost immediately come back out.`

Author: this actually refers to a couple other Saryd students, not a couple as in the relationship term. adding "others" after the "couple" to make it clear, since I forgot apparently

					decline
			`	"Collecting data on population growth, we are. Helping our colleagues, many other ships already are," one of them says.`
			choice
				`	"Alright, I'll show you to your bunks."`
					goto accept
				`	"Well I'm sure you can find someone else then. I'm not looking for any transport missions now."`
					decline
			label accept
			`	They thank you, and start following you to your ship. There are <bunks> of them in total, now that you've had the time to count.`
			`	You ask if this search will stick to Saryd space, and they say that each species usually sticks to their own home territory when gathering this type of data, so you shouldn't worry about having to take them from one end of Coalition space to the other.`
			`	They settle in their bunks, and prepare for the trip.`
				accept
	on complete
		"coalition jobs" ++
		payment 342000
		conversation
			`You're paid <payment> for transporting the Saryds here, and they leave your ship, ready to get to work and ask around.`
			`	They say that they'll probably all be back in the spaceport by night time, so you should check there later if you want to help them further.`

mission "Saryd Census 2"
	name "Saryd Census"

> 	description "Transport these <bunks> Saryds census workers to <destination>, where they will collect data on the population growth there."

Author: Saryds is the plural for them, would it be changed still?

> 	description "Transport these <bunks> Saryds census workers to <destination>, where they will collect data on the population growth there."

Reviewer 3: Yes, because in this case “Saryd” is an adjective describing the workers.

	name "Transport Chikee"
	description "Chikee, the young Kimek who you've transported many times, has finally become a candidate for elevation to the rank of Heliarch. Bring him to <destination> by <date>."
	deadline
	landing
	passengers 1
	to offer
		has "event: chikee break 3"
	source "Ablub's Invention"
	destination "Ring of Wisdom"
	on offer
		conversation
			`You're contacted by Chikee when you land.`
			`	"Captain <last>! Good news, and another job for you, I have," he says, asking that you meet him in the spaceport.`
			`	You head there, and he's already with his bags, prepared to board your ship. "Helped me a lot, House Idriss has. An audience with the Heliarch, on <destination> I have!" he says.`
			`	He says that he needs to be there by <date>, and wastes no time as he immediately heads into your ship.`
				accept
	on complete
		"coalition jobs" ++
		payment 300000
		conversation
			`Chikee hands you <payment> before leaving your ship. Heliarch agents are waiting for him, and receive him in a bit of a ceremony.`
			`	You spend a few hours browsing some shops here, and have some food once you find an establishment that serves something appetizing to humans.`

> 			`	Ringworlds have no day or night, but looking at your ship's internal clock, it is almost night time when Chikee returns to your ship, in a hurry.`

Author: some planets, specifically a few in Coalition space itself, touch on how the day-night cycle works on them, I just added this bit here to be something like that, only in a mission rather than a planet description.

> 			`	Ringworlds have no day or night, but looking at your ship's internal clock, it is almost night time when Chikee returns to your ship, in a hurry.`

Reviewer 4: Fair enough.

> 			`	"Accepted I was! Grant me the rank, they did!" he says, showing you a circlet, a Heliarch circlet, as he puts it on his head. "Infinitely grateful for your help in all of this I am, Captain."`

Author: I'm more inclined to keep that first sentence without a , since there's the usage of a !, so it would be something said in a quick and excited fashion

mission "Coalition Outbreak 1"
	minor
	name "Transport Doctors"
	description "Bring these <bunks> doctors, along with <cargo> to <destination> by <date>, where they will work  to adress a major viral outbreak"
	deadline
	passengers 75
	cargo "Medical Supplies" 90
	to offer
		has "Coalition: Contributor: done"
	source
		near "1 Axis" 6
	destination "Celestial Third"
	on offer
		event "coalition virus contained" 72
		conversation
			`As you enter the spaceport, you notice several battalions of Coalition doctors roaming about, talking with merchant captains and usually then following them to their ships.`

> 			`	One such crowd approaches you, and one of them says, "Broken out on <destination>, a deadly virus has. Captains willing to transport us and <cargo>, we seek." She shows you to the group of doctors, <bunks> of them.`

Author: didn't use that one in specific but should be fixed now

			choice
				`	"I'd be glad to help with something serious like that. Let's go to my ship."`
				`	"I can't afford to fill so many bunks right now, sorry."`
					decline
			`	The doctors rush inside your ship, and spaceport workers quickly place the <cargo> in your cargo hold.`
			`	When you get the chance, you ask them what exactly the virus is, and they explain as best as they can how it causes "tumorous-like behavior on Kimek exoskeletons, rapidly degenerating the tissue and exposing the flesh, all the while increasing the risk of infection". They say that although it is most dangerous to Kimek, there are confirmed cases of some Arachi having contracted the disease.`
			`	It seems pretty serious, so you should make sure these doctors and their supplies are brought to <destination> as quickly as possible.`
				accept
	on visit
		dialog `You've reached <planet>, but the doctors and the medical supplies haven't arrived yet. Depart and wait for the ships carrying them.`
	on complete

> 		payment 768540

Author: I still haven't figured out how to make non-specific payments work so I just use set values

	source "Ki Patek Ka"
	stopover "Fourth Shadow"
	stopover "Blue Interior"
	stopover "Sandy Two"
	stopover "Inmost Blue"
	stopover "Double Haze"
	stopover "Brass Second"
	stopover "Second Viridian"
	stopover "Remote Blue"
	on offer
		conversation
			`This time you're being expected, and are brought to a large conference room where many Saryds, and the occasional Arach, are waiting. A Kimek that is part of the team organizing the containment procedures comes in, and says you will once again be transporting vaccines.`
			`	"In a few hours, fully replicated, the vaccine samples will be. Loaded onto your ships, they will be. Delivered to <stopovers>, they must be."`
			`	Everyone is then separated into groups, and you're put with two Saryd captains, and an Arachi who is comically wearing some protective suit of sorts that covers their entire body, making it somewhat difficult for him to walk.`
			`	The captains agree to follow your lead and deliver the vaccines along with you, so you all head back to your ships.`
				accept
	npc accompany save
		personality escort timid
		government "Coalition"
		ship "Saryd Sojourner" "Noset Temu"
		ship "Saryd Sojourner" "Miest Laba"

> 		ship "Arach Hulk" "Bloobrupblot"

Author: those are Coalition ship names, I just departed and waited for 2 Saryd and an Arach ship and wrote those down

	on stopover
		dialog `You and the Coalition captains have delivered the final batch of vaccines. You must now return to <origin>.`
	on visit
		dialog `You have reached <planet>, but you're missing something! Either you haven't visited <stopovers>, or you left the Coalition ships behind.`
	on complete
		payment 2593230
		"coalition jobs" ++
		conversation
			`You and the other captains are told to wait in one of several lines formed here. As they talk with others and then with you, they mention how many of the captains here just came back from delivering vaccines to Arachi and Saryd space, to the travelers who got infected before going to planets there.`
			`	The line moves quickly, and once it's your turn, you're paid <payment>.`
			`	Looking back at the line, and seeing the sheer number of captains waiting for their share, it's no wonder the Kimek that paid you were extremely brief, but as you come back to your ship, you see that a message was sent to you by the Coalition government. More specifically, it appears to have been sent by their Department of Health.`

> 			`	"To all captains who helped in dealing with the recent viral outbreak, you have our gratitude, and may rest tonight knowing you've contributed to the continued prosperity of our Coalition."`

Author: I'm not sure on the `well`

> 			`	"To all captains who helped in dealing with the recent viral outbreak, you have our gratitude, and may rest tonight knowing you've contributed to the continued prosperity of our Coalition."`

Reviewer 4: Up to you, shouldn't change the sentence in any drastic way.

		require "Jump Drive"
		conversation
			`A Kimek in a lab coat is desperately running around the spaceport here, carrying a small package, with some papers attached.`
			`	As soon as she spots you, she quickly comes to you and says, "Human! The visitor I've heard about, you are! From the outside! A Jump Drive you have, if our space you've reached, yes?"`
			`	You nod, and she continues, "Live on <planet>, a Saryd friend of mine does. Forgotten that his birthday was on <date>, I had! Make it in time for the party, I cannot. Busy with work, I am. At least send his present, I'd like to, but make it in time, regular ships here can't! With your Jump Drive, get there in a day you could. Help me would you, Captain?"`
			choice
				`	"Just that small package? Easy enough, I'll do it."`
					accept
				`	"I'm not headed there, sorry, have another mission to do. Somewhere else."`
					decline
	on complete
		payment 36750
		"coalition jobs" ++
		conversation
			`Apparently this Saryd in question doesn't live here on the spaceport village, so you must take a train to go to his home. Thankfully a train ticket was attached to the package, so you waste no time and get on the next train there.`

> 			`	Even for a world without too large a population, the train wagons here feel unusually empty, with only a handful of other passengers on the one you settle in. The trip takes a few hours, so you have plenty of time to look out the window and gaze at the sights of the seemingly infinite winter forest.`

Author: is the change necessary?

> 			`	Even for a world without too large a population, the train wagons here feel unusually empty, with only a handful of other passengers on the one you settle in. The trip takes a few hours, so you have plenty of time to look out the window and gaze at the sights of the seemingly infinite winter forest.`

Reviewer 4: Nope, just tried to simplify what was being said. Perfectly fine to keep as is.

			`	Once you arrive at the village, you follow the makeshift map of the streets the Kimek included, and deliver the present to her Saryd friend. Apparently his birthday had just started.`
			`	When you get back to your ship, <payment> are transferred to your account, and you get a short message from the Kimek researcher, thanking you for helping out.`

mission "Saryd Students 1"
	minor
	name "Acquire Arachi Rum"
	description "Some Saryd students on <origin> have offered to pay you for bringing them a shipment of traditional Arachi Rum, currently only sold on <destination>."
	deadline
	to offer
		has "Coalition: First Contact: done"
	source "Shadow of Leaves"
	destination "Weir of Glubatub"
	on offer
		conversation
			`A young Saryd bumps into you as you're passing by a nearby shop, knocking you to the ground.`
			`	They help you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`
			`	"No luck?" the translation machine used by the one who bumped into you translates, still active.`
			`	His colleagues shake their heads, "Shipping here, the Arachi no longer are."`

> 			`	Noticing you're still there, the Saryd decides to ask, "Human, a merchant Captain you are, correct? Looking for work, you are?"`

Author: can someone confirm the change? since Captain is always capitalized when used to refer to the player, I did so here.

> 			`	Noticing you're still there, the Saryd decides to ask, "Human, a merchant Captain you are, correct? Looking for work, you are?"`

Reviewer 4: After looking into it, I don't think this one is capitalized. A good rule of thumb might be to ask if `<first> <last>` could be substituted in place of `captain`. That's just something I just came up with, so it might not be correct 100% of the time. If so, it's capitalized, and if not, it's lowercase.

			choice
				`	"What kind of work?"`
					goto work
				`	"Oh no, I live here actually. In the humantown district, right around the corner."`
				`	"Actually I'm just here for the view, sorry."`
					decline

> 			`	They all start laughing, some clearly finding it much funnier than you anticipated. They happen to be the same ones from whom you notice the stronger scent of alcohol.`

Reviewer 4: All good if you don't want to use my substitution here, but I would still recommend some change to the line for the reason stated above.

> 			`	They all start laughing, some clearly finding it much funnier than you anticipated. They happen to be the same ones from whom you notice the stronger scent of alcohol.`

Author: tried redoing it but not sure if I was very successful.

> 			`	"Fair enough. Not a common sight here, humans are, Captain," the one with a translation box says. "Asking for such a response, I was."`

Author: this one for sure is capitalized, you're being directly addressed, and when Captain is used in such a case it's capitalized in the game

> 			`	"Fair enough. Not a common sight here, humans are, Captain," the one with a translation box says. "Asking for such a response, I was."`

Reviewer 4: Yeah, just did some google searching and you're right on this one. Apparently when it's substituted in place of a person's in this way it's a proper noun, and thus capitalized. My bad.

			label work
			`	They tell you that they are students at Starlit University, and will be graduating soon. They intend on throwing a party to celebrate, and had the idea of buying some traditional Arachi Rum for that.`
			`	"Stopped shipping it here, the Arachi have," another Saryd says, pulling on the translation box on his friend's neck, nearly yanking it off. "Happened with many other drinks, this also has."`

> 			`	They explain that the rum is produced on <destination>, and that you'll probably have an easier time finding it there. Though you can't know for sure how many people they intend on inviting to this party, and although you don't know how much a Saryd can drink, you still worry a bit when they say that "two tons should do."`

Author: it read quite weird so I kinda rewrote it, any new issues now?

> 			`	They explain that the rum is produced on <destination>, and that you'll probably have an easier time finding it there. Though you can't know for sure how many people they intend on inviting to this party, and although you don't know how much a Saryd can drink, you still worry a bit when they say that "two tons should do."`

Reviewer 4: Looks good to me!

	destination "Warm Slope"
	on offer
		conversation
			`Apparently you got lucky to have transported a group that was assigned to an area close to the spaceport city, as you see many other Coalition captains impatiently waiting for their own passengers to return, some looking at maps of the planet and grumbling.`
			`	Most of the Saryds are here, but three of them still haven't come back, so the 14 that have already arrived just keep discussing the results they've found.`
			choice
				`	(Go browse some shops, then come back later.)`
					goto shops
				`	(Ask them to tell you about the data they have collected.)`
			`	They mention how this census was mostly focused on the total population growth of the Coalition, and how things are going as expected.`
			`	"Remained stable, the Saryd population has," one of them comments. "At about 30 billion individuals, our numbers still are, though found a very slight increase in fertility rate has been."`
			`	As you continue to chat, one of them gets curious and asks you about humanity's population, to which you respond that your species numbers in about 100 billion people.`
			`	"Double that of the Arachi," one of the workers says.`

> 			`	"And half that of the Kimek," another one completes.`

Author: you can see the exact numbers of each species here https://github.com/endless-sky/endless-sky/issues/2540 , given by MZ himself. you gotta remember that the Coalition also has a complete Quarg ringworld, and 2 incomplete ones. the complete one can hold at least 1 Trillion individuals, so they're better off in total living space than humanity I'd wager

> 			`	"And half that of the Kimek," another one completes.`

Reviewer 4: The scale of some of the stuff always throws me off. Sounds like you're right.

			`	"Only, in need of money for that we are," Inturi says. "Worked in a ship as a crewmember, I did, before decided to stay here and work on ship repairs and maintenance, I had."`
			`	"Much too far away, his old job took him, and well paid crewmembers are not," Aunaris says, bringing up the map again and showing it to you. "Offered a job in Starlit University, on <destination>, I was. Not too distant from <origin>, it is, and a higher pay, it offers. Transport me there, would you, Captain?"`
			choice
				`	"Of course, just come to my ship when you've finished preparing your luggage."`
				`	"Sorry, I'm headed elsewhere now."`
					decline
			`	They both thank you, and you head back to your ship.`
			`	By nightfall, Aunaris arrives, accompanied by Inturi and some other Saryds, who you guess must be family members, as they all say their goodbyes. Once she steps in your ship, you show her to her bunk, and prepare to leave.`
				accept
	on complete
		"coalition jobs" ++
		event "saryd couple has money" 42 86
		conversation

> 			`Aunaris thanks you for bringing her here, and apologizes for not being able to pay you right away, but promises once she and Inturi have figured out their home, they will work hard to pay you.`

Author: added a sentence at the end mentioning the pay will be later. any issues in that new portion?

> 			`Aunaris thanks you for bringing her here, and apologizes for not being able to pay you right away, but promises once she and Inturi have figured out their home, they will work hard to pay you.`

Reviewer 4: Nope, all good.

	source "Shadow of Leaves"
	destination "Far Garden"
	on offer
		conversation
			`You're met with two familiar faces in the spaceport, Inturi and Aunaris, the Saryd couple you met some time ago, who were working on acquring the funding needed to build their own home.`
			`	"Worked for a week in another ship, I did, until landed here we had," Inturi says, joking about his way of catching a ride on a ship without paying a transport fee.`
			`	"Sleeping well, I have not been, but helped much with the campus administration, I have. A bonus, I was given," Aunaris tells you.`
			`	They say they just need a transport to <destination>, the world they have chosen to put down roots, and after everything is done, they will work on getting you your payment.`
			`	You show them to their bunks, and prepare for the trip.`
				accept
	on complete
		"coalition jobs" ++

mission "Saryd Couple 3"
	name "Reversed House Spider"
	description "Take Inturi to <destination>, where he will purchase a small Arach ship that he and Aunaris will repurpose into a home."
	landing
	passengers 1
	to offer
		has "Saryd Couple 2: done"
	source "Far Garden"
	destination "Saros"
	on offer
		conversation

> 			`The couple leaves your ship, and head to a real estate agency. From what they told you during the trip, they have about fifteen hundred thousand credits they can spend, which they claim is in line with the standard properties here, in the most valued (and most sought-after) locations, and that they should still have enough money to pay you afterwards.`

Reviewer 4: ```suggestion
			`The couple leaves your ship and heads to a real estate agency. From what they told you during the trip, they have about fifteen hundred thousand credits they can spend, which they claim is in line with the standard properties in the most valued (and most sought-after) locations here, and that they should still have enough money to pay you afterwards.`
```
Double predicate, no comma. Rearranged for better flow.

> 			`	They return from consulting, the both of them with disappointed looks on their faces. Inturi starts looking at the ships landing and departing, while Aunaris comes talk to you.`

Reviewer 4: ```suggestion
			`	They return from consulting, the both of them with disappointed looks on their faces. Inturi starts looking at the ships landing and departing while Aunaris comes talk to you.`
```
Independent clause followed by dependent clause; no comma.

			`	"Kept watch of one property, for months we had. On a mountain village, overlooking a lake, and with access to a nearby forest, it is. A few days ago, purchased it was," she sighs. "Available, another plot of land neighboring it is, but over half our funds, the land alone costs."`

> 			`	Inturi then looks over, as if he had just received a shock, and says, "That's it! A ship! Live in a ship, we could!"`

Reviewer 4: Creative. I like it.

			choice
				`	"What do you mean?"`

> 				`	"I like the enthusiasm, but unfortunately I'm not looking for any more crewmembers right now."`

Reviewer 4: ```suggestion
				`	"I like the enthusiasm, but, unfortunately, I'm not looking for any more crew members right now."`
```

> 				`	"If you say so. Just don't forget the oranges, scurvy probably is a problem for Saryds too."`

Reviewer 4: Nice little joke/reference.

			`	"Live in a ship?" Aunaris repeats the sentence, puzzled.`

> 			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did, often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy, save time and money otherwise spent building a home, it would."`

Reviewer 4: ```suggestion
			`	"In an Arach ship, for that week I worked. Taught me some things, the captain did. Often repurposed into houses, their ships are," he explains. "Their smallest ship, we could buy. Save time and money otherwise spent building a home, it would."`
```
Both your sentences would be run-on if rearranged back into normal structure.

			`	They talk for a bit in private, and after a few minutes Inturi apparently managed to fully explain his idea and convince her.`

> 			`	They head back to the real estate agency, and arrange the purchase of the plot of land. Inturi then asks that you transport him to <destination>, where the basic Arach ship model is sold, and after buying it he will return here with the new "house."`

Reviewer 4: ```suggestion
			`	They head back to the real estate agency and arrange the purchase of the plot of land. Inturi then asks that you transport him to <destination>, where the basic Arach ship model is sold, and after buying it he will return here with the new "house."`
```
Double predicate; no comma.

> 			`	Aunaris will remain here to finish sorting out all the paperwork, and will be waiting by the property once you two return.`

Reviewer 4: ```suggestion
			`	Aunaris will remain here to finish sorting out all the paperwork and will be waiting by the property once you two return.`
```
Double predicate; no comma.

				accept

mission "Saryd Couple 4"
	name "Test Drive, Final Drive"
	description "Escort Inturi's ship, which will be repurposed into a home for him and Aunaris, to <destination>."
	landing
	to offer
		has "Saryd Couple 3: done"
	source "Saros"
	destination "Far Garden"
	on offer
		conversation
			`Inturi heads right to the shipyard, and you help him with the purchase itself. He says that alhough he has worked in many ships over the years, he never went to the shipyard for more than picking up some parts for repair, or to polish a newly produced model.`

> 			`	Only when you're with him inside the Arach vessel is that you think of asking if he even knows how to fly a ship, but he reassures you that that won't be a problem.`

Reviewer 4: ```suggestion
			`	Only when you're with him inside the Arach vessel do you think of asking if he even knows how to fly a ship, but he reassures you that it won't be a problem.`
```
Replaced a few words to make things cleaner.

> 			`	"To work as a copilot for an old friend, my first job was. Worry about me crashing the ship, you need not," he jokes.`

Reviewer 4: ```suggestion
			`	"Working as a copilot for an old friend, my first job was. Worry about me crashing the ship, you need not," he jokes.`
```
Personal preference.

> 			`	He says he will wait for your lead, and follow you to <destination> once you depart.`

Reviewer 4: ```suggestion
			`	He says he will wait for your lead and follow you to <destination> once you depart.`
```
Double predicate; no comma.

				accept
	npc accompany save
		personality escort timid
		government "Coalition"
		ship "Arach Courier" "Boop"
	on visit
		dialog `You have reached <planet>, but Inturi's ship hasn't arrived yet. Better depart and wait for it to get here.`
	on complete
		"coalition jobs" ++
		event "saryd couple follow-up" 51 96
		conversation
			`You help Inturi park the ship on the plot of land, but since the Arach Courier takes up almost the entire area, your own ship has to land on a regular landing pad.`

> 			`	After taking a transport up the mountain, you see that some workers are already there, and are starting to remove ship components that won't be needed, like its engines and the hyperdrive. A house doesn't need to be able to traverse the void of space, after all.`

Reviewer 4: ```suggestion
			`	After taking a transport up the mountain, you see that some workers are already there and are starting to remove ship components that won't be needed, like its engines and hyperdrive. A house doesn't need to be able to traverse the void of space, after all.`
```
Double predicate; no comma. Removed unnecessary `the`.

> 			`	The couple comes meet up with you, "In a day or two, after some shopping we do, fully furnished the house will be," Aunaris says, looking at the ship.`

Reviewer 4: ```suggestion
			`	The couple comes to meet up with you. "In a day or two, after some shopping we do, fully furnished the house will be," Aunaris says, looking at the ship.`
```

			`	"Helped us a lot in getting it all sorted out you have, Captain <last>. Focus on working hard to pay you back in full, we will," Inturi adds.`
			choice

> 				`	"You don't need to worry about paying me, I helped because I wanted to, not for a payday."`

Reviewer 4: ```suggestion
				`	"You don't need to worry about paying me. I helped because I wanted to, not for a payday."`
```
You might be able to use Amazinite's new payment in conversation feature here. Seems like you have it all worked out here, but just thought I would let you know.

Author: when that is merged

				`	"Well, I hope everything goes well. Best of luck to you two!"`
					goto end
			apply
				set "refused saryd couple payment"

> 			`	They look at you confused and then try to insist on paying you, but after some minutes they agree. "Too kind a heart you are, Captain," Aunaris says.`

Reviewer 4: ```suggestion
			`	They look at you confused and then try to insist on paying you, but after some minutes they agree. "Too kind a heart you are, captain," Aunaris says.`
```

Author: same case as before

			label end
			`	They thank you again, and you head back to your ship.`

event "saryd couple follow-up"

mission "Saryd Couple Follow-Up A"
	invisible
	landing
	to offer
		has "event: saryd couple follow-up"
		not "refused saryd couple payment"
	source
		government "Coalition" "Heliarch"
	on offer
		payment 325400
		conversation
			`When you land, you're notified of <payment> being transferred to your account.`
			`	With it, there's a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`

> 			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there, after people saw their home was a spaceship.`

Reviewer 4: ```suggestion
			`	In the message, they apologize for taking so long to pay you, and go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there after people saw the couple's home.`
```
No comma b/c dependent clause following independent clause. `The couple's home` is a personal preference.

> 			`	They thank you again, and say you're welcome to visit anytime.`

Reviewer 4: ```suggestion
			`	They thank you again and say you're welcome to visit anytime.`
```
Double predicate; no comma.

mission "Saryd Couple Follow-Up B"
	invisible
	landing
	to offer
		has "event: saryd couple follow-up"
		has "refused saryd couple payment"
	source
		government "Coalition" "Heliarch"
	on offer
		conversation

> 			`When you land, see you've received a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`

Reviewer 4: ```suggestion
			`When you land, you see you've received a message. It's from Inturi and Aunaris, the Saryd couple who you helped move out to their very own home.`
```

> 			`	Along with the message, they've sent a picture of them both tending to some flowers and short trees in a little garden they've started up near the ship. In the message they go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there, after people saw their home was a spaceship.`

Reviewer 4: ```suggestion
			`	Along with the message, they've sent a picture of them both tending to some flowers and short trees in a little garden they've started up near the ship. In the message they go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses", as he calls them. Apparently it's become somewhat popular there after people saw the couple's home.`
```
Same edits as above.

> 			`	They thank you again, and say you're welcome to visit anytime.`

Reviewer 4: ```suggestion
			`	They thank you again and say you're welcome to visit anytime.`
```

mission "Longcow Vaccines 1"
	minor
	name "Longcow Vaccines"
	description "Arachi ranchers have asked you to pick up the latest vaccines for longcows on <destination>."
	to offer
		has "Coalition: First Contact: done"
	source
		attributes arach
	destination "New Finding"
	on offer
		conversation
			`You see some Arachi talking among themselves in the spaceport, occasionally glancing at and pointing at you.`

> 			`	One of them finally comes to you, and says, "Greetings, Captain. Interested in performing a cargo mission for us, are you?"`

Reviewer 4: ```suggestion
			`	One of them finally comes to you and says, "Greetings, Captain. Interested in performing a cargo mission for us, are you?"`
```
Double predicate; no comma.

			`	She tells you that they are a cooperative of Arachi ranchers, explains that the safety and health regulations for their business, longcow ranching, were recently updated, and that they need someone to pick up the newest vaccines for the cattle on <destination>.`
			`	"A Jump Drive you have, so get the vaccines faster - and allow us to resume business faster - you would."`
			choice
				`	"You've found the perfect person for the job. I'm in."`
				`	"I suppose that's true, but I'm not looking for courier jobs, sorry."`
					decline

> 			`	The other Arachi come over after she signals to them that you've accepted to help, and they come thank you. They say that after you've picked up the vaccines on <planet> you'll have to bring them to some planets, and wish you a good trip.`

Reviewer 4: ```suggestion
			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the vaccines on <planet>, you'll have to drop them off on another planet before wishing you a good trip.`
```
General edits to fix a couple grammar issues and improve flow.

				accept

mission "Longcow Vaccines 2"
	name "Longcow Vaccines"

> 	description "Take the <cargo> to <destination>, so that the Arachi ranchers can vaccinate their cattle of longcows."

Reviewer 4: ```suggestion
	description "Take the <cargo> to <destination> so that the Arachi ranchers can vaccinate their cattle of longcows."
```
No comma b/c dependent clause follows independent.

	landing
	cargo "Longcow Vaccines" 7
	to offer
		has "Longcow Vaccines 1: done"
	source "New Finding"
	destination "Corral of Meblumem"
	on offer
		conversation

> 			`The dockworkers load <cargo> into your cargo hold, after verifying you've been hired by the Arachi cooperative.`

Reviewer 4: ```suggestion
			`The dockworkers load <cargo> into your cargo hold after verifying you've been hired by the Arachi cooperative.`
```
Double predicate; no comma.

> 			`	A bit after the last crate has been put there, you're contacted by the Arachi, and they say you should bring the vaccines to <destination>, where they will pay you <payment>.`

Reviewer 4: ```suggestion
			`	A bit after the last crate has been placed, you're contacted by the Arachi, and they say you should bring the vaccines to <destination>, where they will pay you <payment>.`
```

				accept
	on complete
		"coalition jobs" ++
		payment 421000

mission "Longcow Vaccines 3"
	name "Longcow Vaccines"
	description "Deliver the remaining <cargo> to <destination>, where some Arachi also work as longcow ranchers."
	landing
	cargo "Longcow Vaccines" 1
	to offer
		has "Longcow Vaccines 2: done"
	source "Corral of Meblumem"
	destination "Nearby Jade"
	on offer
		conversation
			`You're paid 412,000 credits when you land, but as you help the ranchers unload the vaccines, they say that something is off.`

> 			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours is not."`

Reviewer 4: ```suggestion
			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours, is not."`
```
Added comma for reverse-sentence structure consistency.

> 			`	She goes away for about ten minutes, to contact the lab workers back on New Finding. When she comes back, she seemingly jokes with the other ranchers in the Arachi language, before talking to you.`

Reviewer 4: ```suggestion
			`	She goes away for about ten minutes to contact the lab workers back on New Finding. When she comes back, she seemingly jokes with the other ranchers in the Arachi language, before talking to you.`
```
No comma b/c dependent clause follows independent.

			`	"Mixed up the cargo, they have. His own longcow ranch on <destination>, a cousin of mine has," she says. "Sent his batch of vaccines along with my own, they have."`

> 			`	They contact the cousin in question, and he asks that you transport his own part of the vaccines to <destination>, where he will pay you <payment>.`

Reviewer 4: ```suggestion
			`	They contact the cousin in question and he asks that you transport his own part of the vaccines to <destination>, where he will pay you <payment>.`
```
Double predicate; no comma.

				accept
	on complete
		"coalition jobs" ++
		payment 193000
		conversation
			`You land and meet up with the cousin of the Corral of Meblumem rancher, and he pays you <payment>.`
			`	"Mix up our batches due to our family name, the doctors always do!" He grumbles as the vaccines are being removed from your cargo hold.`

mission "Arachi Orphans"
	minor
	name "Arachi Orphans"
	description "Bring these 4 Arachi orphans, their caretaker, and their <cargo> to <destination>, where they will join a foster home."
	passengers 5
	cargo "Orphanage Belongings" 2
	to offer
		has "Coalition: Contributor: done"
	source
		attributes arach
	destination "Chosen Nexus"
	on offer
		conversation

> 			`An elderly Arach beckons you as you enter the spaceport. He is accompanied by four Arachi children, as they all stand close to a large pile of luggage.`

Reviewer 4: ```suggestion
			`An elderly Arach beckons to you as you enter the spaceport. He is accompanied by four Arachi children, who all stand close to a large pile of luggage.`
```

> 			`	He explains that he used to run an orphanage here, but now has gotten too old for the job, and has arranged for the four kids to be transferred to a foster home on <destination>.`

Reviewer 4: ```suggestion
			`	He explains that he used to run an orphanage here, but has now gotten too old for the job. He has arranged for the four kids to be transferred to a foster home on <destination>.`
```
Sentence was long and maybe a run-on.

			`	"Take care of them, the Saryds there will," he says, patting the head of the youngest and shortest of the kids. "Retiring, I am, but abandon them, I will not. An apartment close to the foster home, I will purchase."`

> 			`	He offers to pay you <payment> to bring the five of them, and their <cargo> to <destination>.`

Reviewer 4: ```suggestion
			`	He offers to pay you <payment> to bring the five of them and their <cargo> to <destination>.`
```
Only two items in this list; no comma.

			choice
				`	"Of course I'll help. Here, let me help you get the luggage into my ship."`
					accept

> 				`	"I hope things work out for you, but I'm actually not going to stay in or fly around Coalition space for long now."`

Reviewer 4: ```suggestion
				`	"I hope things work out for you, but I'm actually not going be around Coalition space for long now."`
```
Just seems smoother to me. Your choice.

					decline
	on complete
		"coalition jobs" ++
		payment 268000
		conversation

> 			`You help the old Arach caretaker and some workers take the luggage out of your ship, then head to the orphanage.`

Reviewer 4: ```suggestion
			`You help the old Arach caretaker and some workers take the luggage out of your ship then head to the orphanage.`
```
Double predicate; no comma.

> 			`	A Saryd couple runs the place, and greet you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child, and some Kimek too.`

Reviewer 4: ```suggestion
			`	A Saryd couple runs the place and greets you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child and some Kimek too.`
```
Double predicate; no comma. Also, changed `greet` to singular to match `runs`.
List only has two items; no comma.

> 			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside, to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`

Reviewer 4: ```suggestion
			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`
```
Dependent clause follows independent clause; no comma.

mission "Arachi Cartoonist 1"
	minor
	name "A New Design"

> 	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs for his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."

Reviewer 4: ```suggestion
	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs in his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
```
Changed to avoid `for` used twice close together.

	to offer
		has "Coalition: First Contact: done"
	source "Mebla's Portion"
	destination "Prime"
	on offer
		conversation
			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you on your tracks.`

> 			`	He introduces himself as Drugub, a cartoonist who has seen some local fame lately for his sci-fi comics. "Many ship designs produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`

Reviewer 4: ```suggestion
			`	He introduces himself as Drugub, a cartoonist of local fame for his sci-fi comics. "Many ship designs produced in our space are fascinating, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`
```

Author: his saying "Many ship designs produced in our space are" translates to "Many ship designs are produced in our space", that is, to state that there's what he judges to be a large variety of ships produced there

Reviewer 4: Oh, that makes more sense. You might want to consider rewording that sentence considering I thought it was an error when reading and not just their way of speaking. Could just be me, though.

Author: changed some words to try and clarify

			choice
				`	"I can help with that. Do you have any type of ship in mind?"`
				`	"Sorry, but I won't be heading back to human space for a while."`
					decline
			`	He tells you about how this comic series is set during a less developed time, so he has been trying to take after older ship designs.`

> 			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs, and settles for those.`

Reviewer 4: ```suggestion
			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs and settles for those.`
```
Double predicate; no comma.

			`	He thanks you and wishes you safe travels on your journey to <planet>.`
				accept

mission "Arachi Cartoonist 2"
	name "A New Design"
	description "Return to Drugub on <destination>, and give him the flyer with pictures and some details of Betelgeuse's ship models."
	landing
	to offer
		has "Arachi Cartoonist 1: done"
	source "Prime"
	destination "Mebla's Portion"
	on offer
		conversation

> 			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer, which you then unfold to see all of Betelgeuse's ships depicted, as well as some details on each (and the pricing for each too, of course).`

Reviewer 4: ```suggestion
			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer. Inside, all of Betelgeuse's ships are depicted, and some details on each (along with the pricing, of course) are included.`
```
Long sentence. Edits to improve flow.

Author: didn't follow it exactly. if that caused a new problem do mention it

Reviewer 4: Good, except remove the comma after `depicted`. Independent clause followed by a dependent clause.

			`	"Usually we show these to first time buyers, but you can take one if you want. Are you here for a friend like that? Maybe I could help you figure out what ship would best suit them."`

> 			`	You thank her, saying that your friend does not have the funds for any of the ships, and just wants some info, and head back to your ship.`

Reviewer 4: ```suggestion
			`	You thank her, saying that your friend does not have the funds for any of the ships and just wants some info, then head back to your ship.`
```
Double predicate; no comma.
Thought `then` fits better than `and`, but your choice.

			`	Hopefully a small pamphlet will do for Drugub to get some ideas.`
				accept
	on complete
		"coalition jobs" ++
		payment 75940
		conversation

> 			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's some scroll of knowledge.`

Reviewer 4: ```suggestion
			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's a scroll full of ancient knowledge.`
```
Personal preference.

> 			`	He hands you your payment of <payment>, and spends some time with you still, asking you all the questions he can think of about each ship.`

Reviewer 4: ```suggestion
			`	He hands you your payment of <payment> and spends some time with you still, asking you all the questions he can think of about each ship.`
```
Double predicate; no comma.

			`	"Very beautiful, this 'Leviathan' is. Thinking of many ways to draw it, I already am," he says. "Maybe a flagship of one of my characters, make it I will..."`
			`	He spends some more minutes listening to you after he asks that you describe how you've seen each of the ships perform, thanks you again, and prepares to leave.`
			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan', may I? Just as a small thank you."`
			choice
				`	"Of course! It's called <ship>."`

> 				`	"The name is nothing special, I wouldn't even get the reference of it if I read your comic in the future honestly."`

Reviewer 4: ```suggestion
				`	"The name is nothing special. I wouldn't even get the reference of it if I read your comic in the future, honestly."`
```
Run-on sentence. Not sure if you _need_ a comma for `honestly` or not, but it sounds better to me personally. Up to you.

					goto noship

> 			`	"<ship> it is! Do it justice I will, Captain. Thanks again."`

Reviewer 4: ```suggestion
			`	"<ship> it is! Do it justice, I will, captain. Thanks again."`
```
Added a comma to follow reverse-sentence structure rules.

Author: same as before on Captain

				accept
			label noship

> 			`	"Ah, I see. Well, just a reference it would be, so as you say not very special it would be I suppose. Anyway, thanks again Captain."`

Reviewer 4: ```suggestion
			`	"Ah, I see. Well, just a reference it would be, so as you say, not very special it would be, I suppose. Anyway, thanks again, captain."`
```
Don't really like that middle sentence. You might consider rewriting it in a more concise way.

Author: changed the middle a bit; same thing on Captain as previously mentioned

			`	He tells you how his academic successes have earned him an internship in one of the laboratories on <destination>, and that he needs someone to transport him there by <date> so that he may begin working right away.`
			`	"Better, the payment is, but also hoping for a recommendation letter, I am," he says, explaining that such letters from a Heliarch laboratory could help him significantly in joining their ranks.`
			choice
				`	"Sounds like you've worked pretty hard for this. Let's go to my ship, I'll take you there."`
					goto accept
				`	"You still haven't paid me for last time, though."`
			`	He freezes up for a bit, not responding.`
			`	"Just enough money to rent an apartment there, I have. Save up until the laboratory pays me, I must," he says. "Scamming you, I am not, Captain. Thankful I am, and pay you handsomely, I will."`
			choice
				`	"Alright, I'll wait again. Let's go to my ship."`
					goto accept
				`	"Sorry, but I'm not one to be fooled twice."`
					decline
			label accept
			`	He thanks you, and this time follows you right away, as he already has his bags prepared.`

> 			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he isn't aiming for the more military-focused side of the government.`

Reviewer 5: Instead of saying what he *doesn't* want to do, say what he *does*, and contrast that with what he doesn't.
```suggestion
			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work in the scientific branch of the government, instead of the more military-focused side.`
```

			`	Halfway through, a blizzard makes it tough to see much outside. The ticket inspectors come through again, but now they are serving something akin to hot chocolate, only it is instead a blue, creamy substance.`
			`	Once you arrive at the village, you follow the makeshift map of the streets the Kimek included and deliver the present to her Saryd friend. Apparently the party had just started.`
			`	When you get back to your ship, <payment> are transferred to your account, and you get a short message from the Kimek researcher, thanking you for helping out.`

mission "Saryd Students 1"
	minor
	name "Acquire Arachi Rum"
	description "Some Saryd students on <origin> have offered to pay you for bringing them a shipment of traditional Arachi rum, currently only sold on <destination>. To make it in time, the rum must be bought by <date>."
	deadline
	to offer
		has "Coalition: First Contact: done"
	source "Shadow of Leaves"
	destination "Weir of Glubatub"
	on offer
		conversation

> 			`While walking by some shops, young Saryd bumps into you as you're turning a corner, knocking you to the ground.`
> 			`	He helps you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`

Reviewer 5: ```suggestion
			`While walking by some shops, a young Saryd bumps into you as you're turning a corner, knocking you to the ground. He helps you up, apologizing. A couple others that were with him entered a shop without even noticing you, but almost immediately come back out.`
```

			`	Thankfully, they mention how the census is nearly over and that this was their final destination, so you won't need to wait for hours in the spaceport like the captains you saw on <origin>.`
			`	They thank you for helping them do their job, and wish you safe travels.`

mission "Saryd Couple 1"
	minor
	name "Acquiring Funds"
	description "Inturi and Aunaris, a young Saryd couple, need to make enough money to buy their dream home. Transport Aunaris to <destination>, where she was offered a job in the university there."
	passengers 1
	to offer
		has "Coalition: First Contact: done"
		random < 60
	source
		near "Hunter" 1 2
	destination "Shadow of Leaves"
	on offer
		conversation
			`While walking around the spaceport, you notice one of the Saryd workers that was refueling your ship is quickly galloping somewhere, only without his uniform, as if his shift just ended. He meets up with another Saryd and starts looking at a map with her.`
			choice
				`	(Approach them.)`
				`	(Go back to your ship and check if it was properly refueled.)`

> 					decline

Reviewer 5: I think that this choice should lead to some writing where you find that your ship is fine and you decide to move on with your day, instead of just outright ending here.

Author: added little bit at the end

			`	Along with the message, they've sent a picture of them both tending to some flowers and short trees in a little garden they've started up near the ship. In the message they go on to tell you about how they've been doing there. Aunaris has been working in a school there, while Inturi has been helping other residents set up their own "ship-houses," as he calls them. Apparently it's become somewhat popular there after people saw the couple's home.`
			`	They thank you again and say you're welcome to visit anytime.`

mission "Longcow Antibiotics 1"
	minor
	name "Longcow Antibiotics"
	description "Arachi ranchers have asked you to pick up <cargo> on <stopovers>, then deliver them to <destination>."
	cargo "Longcow Antibiotics" 7
	to offer
		has "Coalition: First Contact: done"
		random < 35
	source
		attributes arach
	stopover "New Finding"
	destination "Corral of Meblumem"
	on offer
		require "Jump Drive"
		conversation
			`You see some Arachi talking among themselves in the spaceport, occasionally glancing at and pointing at you.`
			`	One of them finally comes to you and says, "Greetings, Captain. Interested in performing a cargo mission for us, are you?"`

> 			`	She tells you that they are a cooperative of Arachi ranchers, explains that the safety and health regulations for their business, longcow ranching, were recently updated, and that they need someone to pick up the newest antibiotics for the cattle on <destination>.`

Reviewer 5: ```suggestion
			`	She tells you that they are a cooperative of Arachi ranchers, explains that the safety and health regulations for their business, longcow ranching, were recently updated, and that they need someone to pick up the newest antibiotics for the cattle on <stopovers>.`
```

			`	"A jump drive you have, so get them faster - and allow us to resume business faster - you would."`
			choice
				`	"You've found the perfect person for the job. I'm in."`
				`	"I suppose that's true, but I'm not looking for courier jobs, sorry."`
					decline

> 			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the antibiotics on <planet>, you'll have to drop them off on another planet before wishing you a good trip.`

Reviewer 5: So you've changed this mission to use a stopover, but this wording here is still very roundabout. 
```suggestion
			`	The other Arachi come over to thank you after she signals to them that you've accepted to help. They explain that after you've picked up the antibiotics on <stopovers>, you'll have to drop them off on <planet> before wishing you a good trip.`
```

				accept
	on stopover

> 		dialog `The dockworkers load <cargo> into your cargo hold after verifying you've been hired by the Arachi cooperative. A bit after the last crate has been placed, you're contacted by the Arachi, and they say you should bring the vaccines to <destination>, where they will pay you <payment>.`

Reviewer 5: ```suggestion
		dialog `The dockworkers load <cargo> into your cargo hold after verifying you've been hired by the Arachi cooperative. A bit after the last crate has been placed, you're contacted by the Arachi, and they say you should bring the antibiotics to <destination>, where they will pay you <payment>.`
```

	on visit
		dialog `You land on <planet>, but realize that your escort carrying the longcow antibiotics hasn't entered the system yet. Better depart and wait for it.`
	on complete
		"coalition jobs" += 2
		payment 421000

mission "Longcow Antibiotics 2"
	name "Longcow Antibiotics"
	description "Deliver the remaining <cargo> to <destination>, where some Arachi also work as longcow ranchers."
	landing
	cargo "Longcow Antibiotics" 1
	to offer
		has "Longcow Antibiotics 1: done"
	source "Corral of Meblumem"
	destination "Nearby Jade"
	on offer
		conversation

> 			`You're paid 421,000 credits when you land, but as you help the ranchers unload the vaccines, they say that something is off.`

Reviewer 5: ```suggestion
			`You're paid 421,000 credits when you land, but as you help the ranchers unload the antibiotics, they say that something is off.`
```

> 			`	"Only 6 tons of vaccines, we requested," one of them says. "That extra ton surplus, ours is not."`

Reviewer 5: ```suggestion
			`	"Only 6 tons of antibiotics, we requested," one of them says. "That extra ton surplus, ours is not."`
```

			`	She goes away for about ten minutes to contact the lab workers back on New Finding. When she comes back, she seemingly jokes with the other ranchers in the Arachi language, before talking to you.`
			`	"Mixed up the cargo, they have. His own longcow ranch on <destination>, a cousin of mine has," she says. "Sent along with my own, his batch of antibiotics was."`

> 			`	They contact the cousin in question and he asks that you transport his own part of the vaccines to <destination>, where he will pay you <payment>.`

Reviewer 5: ```suggestion
			`	They contact the cousin in question and he asks that you transport his own part of the antibiotics to <destination>, where he will pay you <payment>.`
```

				accept
	on visit
		dialog `You land on <planet>, but realize that your escort carrying the longcow antibiotics hasn't entered the system yet. Better depart and wait for it.`
	on complete
		"coalition jobs" += 2
		payment 193000

> 		dialog `You land and meet up with the cousin of the Corral of Meblumem rancher, and he pays you <payment>. "Mix up our batches due to our family name, the doctors always do!" he grumbles as the vaccines are being removed from your cargo hold.`

Reviewer 5: ```suggestion
		dialog `You land and meet up with the cousin of the Corral of Meblumem rancher, and he pays you <payment>. "Mix up our batches due to our family name, the doctors always do!" he grumbles as the antibiotics are being removed from your cargo hold.`
```

					accept
				`	"I hope things work out for you, but I'm actually not going to stay in or fly around Coalition space for long now."`
					decline
	on complete
		"coalition jobs" += 3
		payment 268000
		conversation
			`You help the old Arach caretaker and some workers take the luggage out of your ship then head to the orphanage.`
			`	A Saryd couple runs the place and greets you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child and some Kimek too.`
			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`

mission "Arachi Cartoonist 1"
	minor
	name "A New Design"
	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs in his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
	to offer
		has "Coalition: First Contact: done"
	source "Mebla's Portion"
	destination "Prime"
	on offer
		conversation

> 			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you in your tracks.`
> 			`	He introduces himself as Drugub, a cartoonist of local fame for his sci-fi comics. "Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`
> 			choice
> 				`	"I can help with that. Do you have any type of ship in mind?"`
> 				`	"Sorry, but I won't be heading back to human space for a while."`
> 					decline
> 			`	He tells you about how this comic series is set during a less developed time, so he has been trying to take after older ship designs.`
> 			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs and settles for those.`
> 			`	He thanks you and wishes you safe travels on your journey to <planet>.`
> 				accept

Reviewer 5: Got bored. Wrote this.
```suggestion
			`As you're moving along one of the underground tunnels that connect the spaceport here, you notice an Arach off in the distance looking at you. He shouts something in Arachi, then begins running toward you while fiddling with a translation box.`
			`	By the time he reaches you he is moving much slower than when he started. "Vistitor, forgive me," he says, breathing heavily while resting against the nearest wall. "Miss this opportunity to meet you, I could not."`
			`	The Arach takes a deep breath and stands upright. "Drugub, I am. A creator of science fiction comics, I am famous for. Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would. Travel to human space, you may, and return with images for inspirations?"`
			choice
				`	"I can help with that. Do you have any type of ship in mind?"`
				`	"Sorry, but I won't be heading back to human space for a while."`
					decline
			`	Drugub lets out a noise that you would assume is a laugh. "Of the ships of your people, I know not. Please describe them."`
			`	You describe the various shipyards of human space; Lionheart, who develop some of the most advanced ships in human space; Tarazed, who design their ships with inspiration from the Quarg (resulting in a noticeable grimace from Drugub); Syndicate, who specialize in cheap yet (somewhat) effective ships.`
			`	After describing the ships of Betelgeuse Shipyards on <planet> and how some of their designs are almost as old as humanity's hyperdrive, he stops you. "Perfect, this would be. Set during a... less developed time, my series is. Older designs, I have been searching for."`
			choice
				`	"Okay. I'll look for something to bring back for you."`
					goto end
				`	"Sounds like a plan. Let's get ready to leave."`
			`	"Us?" Drugub asks, letting out another noise that you think is a laugh. "No, only you may go. Leave Coalition space, the Heliarchs do not allow. Permission, I require. Without me, you must go. Bring images for me is all I need."`
			label end
			`	He thanks you and wishes you safe travels on your journey to <destination>.`
				accept
```

	on offer
		set "license: Coalition"
		fail

mission "Kimek Student 1"
	minor
	name "Transport Chikee"
	description "Chikee, a young Kimek, is studying to become a Heliarch. Transport him to <destination>, where he was offered a job, so that he can make a living while studying on his own."
	passengers 1
	to offer
		has "Coalition: First Contact: done"
		random < 90
	source "Fourth Shadow"
	destination "Second Rose"
	on offer
		conversation
			`At any given time there are thousands of Kimek here in the spaceport, either mingling among themselves during a break from work or negotiating travel prices with starship captains. You imagine that if you weren't a human and clearly stood out, you too would be constantly getting offers to travel to another world, as many civilian captains here wish to fill a last bunk or two they have vacant on their ship before departing.`

> 			`	You manage to find a restaurant here where you spot some food you've already learned you can eat, so you take a seat and order something.`

Author: dunno, this may well be the last mission a player ever comes across in Coalition space, and by that point if they have been only in Coalition space doing missions and taking jobs for some ingame months it'd be odd to have this one say "you finally decide to try the food here"

			`	You're halfway through your meal when you notice the waiter who brought it to you still hasn't moved away from the table.`
			choice
				`	"Can I help you?"`
				`	"Sorry, should I have paid first before eating?"`
			branch translator
				has "language: Coalition"
			`	The Kimek doesn't have a translation box, so your words don't reach him. While he insists on remaining by the table, you also notice he has been constantly glancing at the entrance, as if waiting for someone to come in.`
			`	When a Heliarch agent passes by the entrance, the Kimek says something in his language, heading to meet the Heliarch, and speaking with them briefly. The Heliarch agent then accompanies the Kimek waiter to your table.`
			choice
				`	"Is there an officer problem?"`

> 				`	"Did I do something wrong?"`

Author: just removed the joke option and replaced with another neutral one

			`	"In a few hours, fully replicated, the vaccine samples will be. Loaded onto your ships, they will be. Delivered to <stopovers>, they must be."`
			`	Everyone is then separated into groups. You're put with two Saryd captains and an Arachi who is wearing some protective suit of sorts that covers their entire body, making it somewhat difficult for him to walk.`
			`	The captains agree to follow your lead and deliver the vaccines along with you, so you all head back to your ships.`
				accept
	npc accompany save
		personality escort timid
		government "Coalition"
		ship "Saryd Sojourner" "Noset Temu"
		ship "Saryd Sojourner" "Miest Laba"
		ship "Arach Hulk" "Bloobrupblot"
	on stopover
		dialog `You and the Coalition captains have delivered the final batch of vaccines. You must now return to <origin>.`
	on visit
		dialog `You have reached <planet>, but you're missing something! Either you haven't visited <stopovers>, or you left the Coalition ships behind.`
	on complete
		payment 2593230
		"coalition jobs" ++
		conversation

> 			`You and the other captains are told to wait in one of several lines formed here. As they talk with others and then with you, they mention how many of the captains here just came back from delivering vaccines to Arachi and Saryd space, to the tourists who contracted the disease before flying to those areas.`

Author: redid to try and make it readable

			`	The line moves quickly, and once it's your turn, you're paid <payment>.`

> 			`	Looking back at the line, and seeing the sheer number of captains waiting for their share, it's no wonder the Kimek that paid you were extremely brief. As you come back to your ship, you see that a message was sent to you by the Coalition government. More specifically, it appears to have been sent by their Department of Health.`

Author: added thing

			`	Even for a world without too large a population, the train wagons here feel unusually empty, with only a handful of other passengers on the one you settle in. The trip takes a few hours, so you have plenty of time to look out the window and gaze at the sights of the seemingly infinite winter forest.`
			`	Halfway through, a blizzard makes it tough to see much outside. The ticket inspectors come through again, but now they are serving something akin to hot chocolate, only it is instead a blue, creamy substance.`
			`	Once you arrive at the village, you follow the makeshift map of the streets the Kimek included and deliver the present to her Saryd friend. Apparently his birthday had just started.`
			`	When you get back to your ship, <payment> are transferred to your account, and you get a short message from the Kimek researcher, thanking you for helping out.`

mission "Saryd Students 1"
	minor
	name "Acquire Arachi Rum"
	description "Some Saryd students on <origin> have offered to pay you for bringing them a shipment of traditional Arachi Rum, currently only sold on <destination>."

> 	deadline

Author: added

	to offer
		has "Coalition: First Contact: done"
		random < 80
	source "Shadow of Leaves"
	destination "Weir of Glubatub"
	on offer
		conversation
			`A young Saryd bumps into you as you're passing by a nearby shop, knocking you to the ground.`
			`	He helps you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`
			`	"No luck?" the translation machine used by the one who bumped into you translates, still active.`
			`	His colleagues shake their heads, "Shipping here, the Arachi no longer are."`

> 			`	Noticing you're still there, the Saryd decides to ask, "Human, a merchant captain you are, correct? Looking for work, are you?"`

Author: did thing to change it up, added a branch so that you do get their words in case the player has the Coalition language

			choice
				`	"Of course I'll help. Here, let me help you get the luggage into my ship."`
					accept
				`	"I hope things work out for you, but I'm actually not going to stay in or fly around Coalition space for long now."`
					decline
	on complete
		"coalition jobs" ++
		payment 268000
		conversation
			`You help the old Arach caretaker and some workers take the luggage out of your ship then head to the orphanage.`
			`	A Saryd couple runs the place and greets you and the old Arach when you get there. Aside from a few Saryd kids, you notice the place already has an Arach child and some Kimek too.`
			`	After you make sure all their belongings are inside the foster house, the Arachi children head on inside to get acquainted with their new family. Their caretaker thanks you and hands you <payment> before heading inside himself.`

mission "Arachi Cartoonist 1"
	minor
	name "A New Design"
	description "The Arach cartoonist Drugub is currently struggling to find inspiration for new spaceship designs in his comics. Head to <destination> and find something you can bring back to show him the ship models produced there."
	to offer
		has "Coalition: First Contact: done"

> 		random < 95

Reviewer 5: You can remove this.

	source "Mebla's Portion"
	destination "Prime"
	on offer

> 		conversation

Reviewer 5: I feel like this conversation could stand to be improved. Right now as it is, there seems to be little introduction of this character. Perhaps he shows you some of his comics to get you interesting. Then when you say you're interested in helping, it seems to me like you would be bringing him to human space, but instead we get a "Okay, that sounds good. Good luck traveling across the galaxy for me!" Maybe instead bring him as a passenger to see the ships first hand. If he can't leave Coalition space, perhaps mention something about not being allowed to leave by the Heliarchs, only asking that you take pictures or get a hologram to show to him.

> 			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you on your tracks.`

Reviewer 5: ```suggestion
			`As you're moving along one of the underground tunnels that connect the spaceport here, an Arach stops you in your tracks.`
```

			`	He introduces himself as Drugub, a cartoonist of local fame for his sci-fi comics. "Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "Expand my creative horizon, seeing the general shape of some human ships would."`
			choice
				`	"I can help with that. Do you have any type of ship in mind?"`
				`	"Sorry, but I won't be heading back to human space for a while."`
					decline
			`	He tells you about how this comic series is set during a less developed time, so he has been trying to take after older ship designs.`
			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs and settles for those.`
			`	He thanks you and wishes you safe travels on your journey to <planet>.`
				accept

mission "Arachi Cartoonist 2"
	name "A New Design"
	description "Return to Drugub on <destination>, and give him the flyer with pictures and some details of Betelgeuse's ship models."
	landing
	to offer
		has "Arachi Cartoonist 1: done"
	source "Prime"
	destination "Mebla's Portion"
	on offer
		conversation

> 			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer. Inside, all of Betelgeuse's ships are depicted with some details on each (along with the pricing, of course).`

Reviewer 5: ```suggestion
			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships," she hands you a flyer. Inside, all of Betelgeuse's ships are depicted with some details on each (along with the pricing, of course).`
```

> 			`	"Usually we show these to first time buyers, but you can take one if you want. Are you here for a friend like that? Maybe I could help you figure out what ship would best suit them."`

Reviewer 5: ```suggestion
			`	"Usually we show these to first time buyers, but you can take one if you want. Is your friend interested in buying? Maybe I could help you figure out what ship would best suit them."`
```

			`	You thank her, saying that your friend does not have the funds for any of the ships and just wants some info, then head back to your ship.`
			`	Hopefully a small pamphlet will do for Drugub to get some ideas.`
				accept
	on complete
		"coalition jobs" ++
		payment 75940
		conversation
			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's some scroll of knowledge.`
			`	He hands you your payment of <payment> and spends some time with you still, asking you all the questions he can think of about each ship.`
			`	"Very beautiful, this 'Leviathan' is. Thinking of many ways to draw it, I already am," he says. "Maybe a flagship of one of my characters, make it I will..."`
			`	He spends some more minutes listening to you after he asks that you describe how you've seen each of the ships perform, thanks you again, and prepares to leave.`

> 			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan', may I? Just as a small thank you."`

Reviewer 5: ```suggestion
			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan,' may I? Just as a small thank you."`
```

	to offer
		has "Coalition: First Contact: done"
	source "Mebla's Portion"
	destination "Prime"
	on offer
		conversation
			`As you're moving along one of the larger underground tunnels that connect the spaceport here, a dozen makeshift stands, each with a different apparent 'theme' to them, captures your attention. When you get closer, it's clear that the event is something of a convention, each stand managed by a different artist that lives here.`
			`	After you're done looking at a stand that showcased various fantasy attires and move on to the neighboring one, it's owner jumps as soon as he turns to see you, looking dumbfounded. He introduces himself as Drugub, a cartoonist of local fame for his science fiction comics, showing various examples of them as he speaks with you. "Many styles of ships produced in our space are, but only so many ideas from them, I can get," he says. "That you came here, lucky I am. To expand my creative horizon, I need, and seeing the general shape of some human ships would."`
			choice
				`	"I can help with that. Do you have any type of ship in mind?"`

> 				`	"Sorry, but I won't be heading back to human space for a while."`

Reviewer 5: I'd change this second choice to not mention going to human space, as it isn't immediately apparent to me that that is what Drugub is asking. (It could be that he just wants to see my ship.)

Author: changed it up

					decline

> 			`	He looks puzzled, "For a variety of purposes, ships in my comics are used, so to know of a variety of ships, preferrable it would be."`

Reviewer 5: ```suggestion
			`	He looks puzzled. "For a variety of purposes, ships in my comics are used, so to know of a variety of ships, preferable it would be."`
```

			`	You decide to give him a brief description of the biggest shipyards in human space, citing their history and design philosophy, from Syndicated Shipyards' cheaper designs to Deep Sky's sophisticated (by human standards) technology.`
			`	When you're done, he thinks for a bit, then concludes that Betelgeuse Shipyards' older, historical designs, are what he's looking for. "Set during a less developed time, what I'm working on is," he explains.`
			choice
				`	"In that case, I'll head to their home shipyard and see what I can bring back for you."`
					goto end

> 				`	"Alright, it's a bit of a long trip to <planet> but if you're sure, let's head to my ship and I'll show you your bunk."`

Reviewer 5: ```suggestion
				`	"Alright, it's a bit of a long trip to <planet>, but if you're sure, let's head to my ship and I'll show you your bunk."`
```

					decline
			label accept
			`	He thanks you, and this time follows you right away, as he already has his bags prepared.`

> 			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work for for the scientific branch of the government, instead of the more military-focused side.`

Reviewer 5: ```suggestion
			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work for the scientific branch of the government, instead of the more military-focused side.`
```

	destination "Weir of Glubatub"
	on offer
		conversation

> 			`While walking by some shops, young Saryd bumps into you as you're turning a corner, knocking you to the ground. He helps you up, apologizing. A couple others that were with him entered the shop without even noticing you, but almost immediately come back out.`

Reviewer 5: ```suggestion
			`While walking by some shops, a young Saryd bumps into you as you're turning a corner, knocking you to the ground. He helps you up, apologizing. A couple others that were with him entered a shop without even noticing you, but almost immediately come back out.`
```

			`	"Much too far away, his old job took him, and well paid crewmembers are not," Aunaris says, bringing up the map again and showing it to you. "Offered a job in Starlit University, on <destination>, I was. Not too distant from <origin>, it is, and a higher pay, it offers. Transport me there, would you, Captain?"`
			choice
				`	"Of course, just come to my ship when you've prepared your luggage."`
				`	"Sorry, I'm headed elsewhere now."`
					decline
			`	They both thank you, and you head back to your ship.`
			`	By nightfall, Aunaris arrives, accompanied by Inturi and some other Saryds, who you guess must be family members, as they all say their goodbyes. Once she steps in your ship, you show her to her bunk, and prepare to leave.`
				accept
	on complete
		"coalition jobs" ++
		event "saryd couple has money" 42 86
		conversation
			`Aunaris thanks you for bringing her here, and apologizes for not being able to pay you right away, but promises once she and Inturi have figured out their home, they will work hard to pay you.`

> 			`	"Stay here for a few months, I probably will, before the remaining funds I have," she says, before saying goodbye and taking a cable car to her new workplace.`

Author: that'd get her sentence to `"Stay here for a few months, I probably will, before the remaining funds, I will have"` which sounds somewhat off to me. added an `acquired` at the end instead, does that work too?

			`	You thank her, saying that your friend does not have the funds for any of the ships, and just wants some info, and head back to your ship.`
			`	Hopefully a small pamphlet will do for Drugub to get some ideas.`
				accept
	on complete
		"coalition jobs" ++
		payment 75940
		conversation
			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's some scroll of knowledge.`
			`	He hands you your payment of <payment>, and spends some time with you still, asking you all questions he can think of about each ship.`
			`	"Very beautiful, this 'Leviathan' is. Thinking of many ways to draw it, I already am," he says. "Maybe a flagship of one of my characters, make it I will..."`
			`	He spends some more minutes listening to you after he asks that you describe how you've seen each of the ships perform, thanks you again, and prepares to leave.`
			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan', may I? Just as a small thank you."`
			choice
				`	"Of course! It's called <ship>."`

> 				`	"The name is nothing special, I wouldn't even get the reference it if I read your comic in the future honestly."`

Reviewer 6: seems like you forgot an "of" before "it" here :
>"I wouldn't even get the reference it if"

into : "I wouldn't even get the reference of it if"
